client/css/common.css

```diff
@@ -0,0 +1,89 @@
+/* Overwriting Bootstrap CSS */
+.nav-link {
+    color: #333;
+}
+
+.nav-link.active {
+    color: #007bff;
+}
+
+.row {
+    margin-left: 0px !important;
+    margin-right: 0px !important;
+}
+
+/* Top bar */
+#top-bar {
+    padding-bottom: 0px;
+    padding-top: 0px;
+}
+
+/* Sidebar */
+#sidebar {
+    width: inherit;
+    min-width: 220px;
+    max-width: 220px;
+    background-color: #f5f5f5;
+    opacity: 1;
+    float: left;
+    height: 100%;
+    position: relative;
+    overflow-y: auto;
+    overflow-x: hidden;
+    z-index: 10; /* in front of overlay */
+
+    /* Smooth transition in and out */
+    -webkit-transition: margin .25s ease-out;
+    -moz-transition: margin .25s ease-out;
+    -o-transition: margin .25s ease-out;
+    transition: margin .25s ease-out;
+}
+
+#sidebar.active {
+    margin-left: -250px;
+}
+
+#sidebar-overlay {
+    display: none;
+    position: fixed;
+    width: 100vw;
+    height: 100vh;
+    background: rgba(0, 0, 0, 0.7);
+    z-index: 9; /* behind actual sidebar */
```

Comment:
See above comment about stance of z-index. (Imagine having to balance 100s of these)